### PR TITLE
test/params_test.c: diverse fixes

### PR DESCRIPTION
**Pull request by levitte**
79c0b33 test/params_test.c: make construct_api_params() static
9eaf623 test/params_test.c: make more global variables static
f2003c8 test/params_test.c: use TEST_double_eq to check doubles

### CODE REVIEW

File: test/params_test.c [hunk 1]
```diff
@@ -491,7 +491,7 @@ static int test_case_variant(const OSSL_PARAM *params,
         struct object_st *sneakpeek = obj;
 
         if (!TEST_int_eq(sneakpeek->p1, app_p1)         /* app value set */
-            || !TEST_ulong_eq(sneakpeek->p2, p2_init) /* Should remain untouched */
+            || !TEST_double_eq(sneakpeek->p2, p2_init) /* Should remain untouched */
```
Comment (romen) [inline test/params_test.c]: nit: align the comment

**Review by romen — APPROVED** [review]
Just a nit!

File: test/params_test.c (hunk 1 above)
Comment (levitte) [reply test/params_test.c]: Sone

**Comment by levitte** [comment]
I'll just wait for the CIs to confirm as well...

**Comment by levitte** [comment]
Cis happy.  Merged.

d3620841cc39aaa4a3d75ea32e94ccd110b5bef4 test/params_test.c: use TEST_double_eq to check doubles
932c3d0f6706a08932d89c8610b06cb57fba13ce test/params_test.c: make more global variables static
847275075f971afeb5fa928c078edf3bc17cddb3 test/params_test.c: make construct_api_params() static

**Pull request closed without merging**